content/docs/welcome-guide/create/creating-projects-using-cli.md

```diff
@@ -113,9 +113,14 @@ If your project build directory is outside the project path, you must include th
         ```
 
         {{< important >}}
-If you built the engine from source using the `INSTALL` target, make sure that you launch the Editor and other tools from the install directory, _not_ the engine's build directory. The Windows install directory typically ends in `/bin/Windows/profile`.
+If you built the engine from source using the `INSTALL` target, make sure that you launch the Editor _and_ other tools from the installed engine's build directory, _not_ the engine's build directory. The Windows install directory typically ends in `/bin/Windows/profile`.
```

Comment:
Possible reword for clarity - If you built the engine from source using the `INSTALL` target, make sure that you launch the Editor _and_ other tools from the directory where the engine is installed, _not_ the directory where 
 the engine was built from. The Windows install directory typically ends in `/bin/Windows/profile`.

Reply:
This reword introduces a passive voice, which we try to avoid in these docs. I'm inclined to prefer the current proposed change.